app/src/main/java/com/x8bit/bitwarden/data/tools/generator/repository/GeneratorRepositoryImpl.kt

```diff
@@ -54,7 +54,7 @@ class GeneratorRepositoryImpl(
     private val authDiskSource: AuthDiskSource,
     private val vaultSdkSource: VaultSdkSource,
     private val passwordHistoryDiskSource: PasswordHistoryDiskSource,
-    private val policyManager: PolicyManager,
```

Comment:
This removal seems out of place. Is it intentional?

Reply:
it was not being used but if we would prefer to do this in a another PR or if there is some future use for this that I am not aware I can leave it.

Reply:
Nah, just making sure it was intentional. 👍